crates/rattler_virtual_packages/src/lib.rs

```diff
@@ -130,11 +175,15 @@ impl From<VirtualPackage> for GenericVirtualPackage {
 impl VirtualPackage {
     /// Returns virtual packages detected for the current system or an error if the versions could
     /// not be properly detected.
-    pub fn current() -> Result<&'static [Self], DetectVirtualPackageError> {
-        static DETECTED_VIRTUAL_PACKAGES: OnceCell<Vec<VirtualPackage>> = OnceCell::new();
-        DETECTED_VIRTUAL_PACKAGES
-            .get_or_try_init(try_detect_virtual_packages)
-            .map(Vec::as_slice)
+    pub fn current() -> Result<Vec<Self>, DetectVirtualPackageError> {
+        try_detect_virtual_packages_with_overrides(&VirtualPackageOverrides::default())
```

Comment:
I think this is a breaking change because now all of a sudden environment variables can override the previous behavior. I think we should restrict the behavior to `current_with_overrides` or perhaps deprecate this function.

We can also rename `current_with_overrides` to `detect`?